servicebroker_node/test/fat/index_test.js

```diff
@@ -0,0 +1,34 @@
+var supertest = require("supertest");
```

Comment:
You have units which test everything.
Breaking things down provides no additional value other than unnecessary setup that should be done within the tests anyhow.

Integration tests are exactly that. They test the integration between components which are outside the scope of a single project. It would be similar to where acceptance tests live which requires a live backend. Integration tests create a real setup but not within the target environment,